crates/rpc/rpc/src/eth/cache/mod.rs

```diff
@@ -77,6 +99,7 @@ impl EthStateCache {
             full_block_cache: BlockLruCache::new(max_blocks, "blocks"),
             receipts_cache: ReceiptsLruCache::new(max_receipts, "receipts"),
             evm_env_cache: EnvLruCache::new(max_envs, "evm_env"),
+            fee_history_cache: FeeHistoryLruCache::new(max_blocks, "fee_history"),
```

Comment:
we don't need this here, all of this should be done by a standalone feehistory type, similar to gaspriceoracle